apps/modernization-ui/src/apps/page-builder/pages/ManagePages/ManagePages.scss

```diff
@@ -5,8 +5,35 @@
     .manage-pages__container {
         display: flex;
         flex-direction: row;
-        position: absolute;
-        top: 54px;
+    }
+    .manage-pages__header {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        button {
+            padding: 9px 16px;
```

Comment:
Should we be using REM?

Reply:
I do use REM as a general rule--but things like button padding seems like a more fixed/exact situation.